common/lib/protocol-definitions/CHANGELOG.md

```diff
@@ -1,5 +1,30 @@
 # @fluidframework/protocol-definitions Changelog
 
+## [3.2.0](https://github.com/microsoft/FluidFramework/releases/tag/protocol-definitions_v3.2.0)
+
+### Added new optional targetClientId property to ISequencedClient
+
+`ISequencedClient` has a new optional property called `targetClientId` which is the client ID of the singular client the
+signal is being (or has been) sent to.
+
+([#19555](https://github.com/microsoft/FluidFramework/issues/19555)) [af2d678db1](https://github.com/microsoft/FluidFramework/commits/af2d678db1216475a444bd35354c8284e6729973)
+
+### Added new optional omitted property to ISnapshotTree
+
+`ISnapshotTree` has a new optional property, `omitted`. This property can be used by the service to communicate that
+they omitted the blobs contents in the snapshot.
```

Comment:
Use "can" here is a bit concerning.
Is this required to be set if and only if the server omitted blobs?
Do older clients support this case? If not, how do they inform the service that they are optining into blobs being able to be omitted?

Also I think the service should be "it" not "they".

Reply:
The original change description in the PR used "will" instead of "can." Updated.

For the other questions about how these changes will roll out safely across clients and servers, I am not sure. @jatgarg @vladsud - can you help clarify?

Reply:
Interfaces only describe protocol capabilities. They do not describe / define when it's safe for either client or service to change behavior and use these capabilities. When it comes to snapshots, new capability (ability to omit part of the tree on download) will be enabled only when all of these items land:
1. Loader changes propagate through all existing clients to reasonable saturation (this is less about correctness and more about performance, as old loader will always ask for full snapshot and service will provide one in back compat manner).
2. ODSP service changes propagate to 100% saturation in PROD.
3. runtime changes propagate to 99.99% saturation (we never reach 100% here, even after 2 years)

Only when all these conditions are met, particular feature in particular client (Loop & row-as-pages is our first consumer) will start using new capability to instruct service to exclude certain parts of the snapshot on download.

One key here - protocol changes and file format changes always happen independently from FF versions. All such changes always ship dark, and are enabled when proper criteria are met, either through feature gates, or through further code deployment that enables such feature.